documentation/managing_modules.md

```diff
@@ -0,0 +1,183 @@
+# 🧪 Managing modules in Bolt projects
```

Comment:
I also wonder if adding a "Why we did this" or "Benefits of using this workflow" type of section would be useful. No more accidentally wiping modules in`modules/`, no more having to manually list and resolve dependencies in the Puppetfile.

Reply:
I can put that in for now, but I think that's the kind of info that should go in a developer update (perhaps along with the "What's changed" stuff). Reason being, I'll probably remove a lot of that sort of info once this properly replaces the old way of doing things.

Reply:
Also, this is kind of what happened with Projects - I like the idea of a more permanent home for "Why we're doing stuff"